euphoria-service/src/main/java/edu/cooper/ece366/euphoria/PostingHandlers.java

```diff
@@ -203,12 +200,13 @@ public PostingHandlers(final ObjectMapper objectMapper, final Config config) {
     @VisibleForTesting
     public List<Posting> createPosting(final RequestContext rc) {
         try {
-            Integer companyId = Integer.valueOf(rc.pathArgs().get("companyId"));
-            String jobTitle = rc.pathArgs().get("jobTitle");
-            String description = rc.pathArgs().get("description");
-            Location location = Location.valueOf(rc.pathArgs().get("location"));
-            Industry industry = Industry.valueOf(rc.pathArgs().get("industry"));
-            SkillLevel skillLevel = SkillLevel.valueOf(rc.pathArgs().get("skillLevel"));
+            Map jsonMap = objectMapper.readValue(rc.request().payload().get().toByteArray(), Map.class);
+            Integer companyId = (Integer) jsonMap.get("companyId");
```

Comment:
change to 
Integer companyId = Integer.parseInt(jsonMap.get("companyId").toString());

Reply:
:+1: